### ISSUE TITLE
As A Conference Planner I would like to see time to prepare presentation only in hours

### ISSUE THREAD

**Issue by fullstackto** [kosamari/who-pays-tech-speakers#12]
Only allow submission of presentation prep time in hours, highly unlikley some people took a full 3 weeks to prep, I am analyzing the data assuming 1 hour a day.

**Comment by kosamari** [comment]
@fullstackto Thank for suggestion! 

Here is my original thought when designing the form.
- focus on revealing (usually) untold story of speaking
- make is as simple as possible to get many response  

Because of that collecting accurate comparable data was 2nd priority. This is also why the site shows story not data visualization.

I didn't think people are interested in prep time so much ! so I'd like to see what other people wants to hear :)

**Comment by sole** [comment]
nope. some talks take MONTHS to prepare, specially if they involve DEMOS. Demos are what literally devours all your time for talk prep.

**Comment by cczona** [comment]
I can see a case to be made for hours vs days/etc. I suspect few people set aside multiple full work days for talk prep. I assume the common case is piecemeal as time allows. In a twitter conversation a year or two ago, someone asked people how long their prep was. If I recall, most/all got expressed in hours.

If we want to run comparison by hour, there's also the complication that not everyone may mean the same number of hours when they say "day" etc. Is 8 hour day universal outside the US? And would weeks be assumed to be 5 days or 7 days?

+1 for this issue. One less dropdown (UX win), and greater clarity of data (analysis win).

**Comment by KatieK2** [comment]
When I prep for a talk, it usually takes up _all the spare time I can give it_. This means most bus rides, lunches, evenings and full days on weekends. It can be a very significant burden, especially for a new talk (newer speakers have more new talks) and when I've only got a month or two of notice for the conf. If I've noted x days to prep for a talk, estimate 4 or 5 hours average per day.

When I have a talk accepted, my new hobby (to the near-exclusion of other hobbies) is writing that talk. Preparation involves researching by reading books and web materials, watching other videos; creating and browser-testing demos; writing blogs; sourcing, creating and clipping images; writing, practicing and updating the slides. 

I do think there is interest in knowing how much time it takes to prep for a talk. I've been asked about that more than once at confs. And prep time varies _hugely_ - I talked to one speaker who said they spent about 3 evenings in the week before the conf preparing.

I see a trade-off between getting the most responses by allowing multiple forms of time measurement, and getting "reliable" data. "Hours" is probably more accurate, but I'd have a much harder time estimating that. I suspect that everyone would be estimating hours with a large delta, and thus "hours" would be difficult to quantify anyway.

**Comment by kosamari** [comment]
Thank you all for sharing thought ! I'm going to close this issue and keep the form as is. 

Primally, the intent of the project is about sharing story not collecting measurable data. So I'm taking approach of "what ever people want to describe their process." Some might round up 8 hours of work as 1 day, some say 3 weekends of work as 6 days or 3 weeks that's up to them. 

Also, here is some data to make this decision.
-  Out of 164 answers to this question, 120 used "days" as unit
  - out of that 120 answers, 99 answers stated less than 7 days (so didn't use other unit like weeks)
  - out of that 99 answers, 68 answers stated more than 3 days.  I'm no expert, but cognitively if it took me more than a weekend to prepare for a talk, it's easier to describe in bigger unit.

# 

@fullstackto 

> I am analyzing the data assuming 1 hour a day.

Now, as speaker I must admit conference organizer making this assumption is incredibly worrisome.
Here is full break down of what went into my recent conference talk at RejectJS 2015.
- 2 full days of hardware demo fabrication (with in-kind help from partner & 2 friends)
- 2 full days I was off from work to code demo app (specifically for this conference)
- 2 full days of research trip I made to museums and meeting people for interviews (used as reference in my talk)
- 3 full days of slide making (one full weekend for index carding, white boarding, taking pictures, and 10 hours of making slides - I know specific hours because I finished in one night & remember start and end time)

This already add up to 9 days, not to mention over few months at nights & weekends improving my project thinking "Oh, adding this feature and making demo at a conference would be nice!". 
I also took 1 week off from work to fully focus on my project this summer. Which all of the work from that week ended up in my talk.

So I hope you add more consideration when reading data from this project.

**Comment by cczona** [comment]
@kosamari I just want to note that even though stories are your primary intent, not measurable data, that you were able to cite a bunch of measured data here is testimony to how much having measurable and accurate data is valuable.

**Comment by kosamari** [comment]
@cczona oh yes, I love data <3  my analysis is about "which unit is most used & how it's used" which is core part of designing how to tell individual story. 
If I want to make a full dataset to do analysis, I would design different site.

**Comment by cczona** [comment]
Gotcha.

I'm wary that folks (whether providing date or reading is) apply different assumptions to the number of hours implied by "day" or "week".

Is it likely that someone's prep is measurable in less than 60 minutes? Seems to me that one is safe to drop, and let someone round it to 1 hour.

**Comment by fullstackto** [comment]
Hi @kosamari I'm sorry you feel worried about our organizing. We are a non-profit and fall into the "no-budget" so I had to devise a way of reading the data of prep time to try and come up with a baseline of how to pay speakers.

To pay a speaker just for the time I know it takes them, the 60 minute slot we give them wouldn't be an accurate measure for compensations.

Now when I said I assumed 1day=1hour that was not set in stone, I added a bit of fabrication to the numbers when converting everything to minutes.

In instances where the data showed <7 days in prep time a day was set to 1-3 hrs, when data was between 7+ days but under a month I made that about 28 hours, for instances where the data set was large it got tricky, how much times does one spend if the say they took 1-3 months.

In that case I went with the assumption these talks are being developed in their free time, which is normally 8 hours a day and only assuming 5 days in the week, as some probably also have families and other commitments. For these I put 48 to 144hrs.

This of course is all variable and why I use the term average. Now that the data set is twice as large I am going to run the numbers again and see what I get :)

I developed this baseline compensation table from the first iteration of the data, assuming a freelancer rate of $75 an hour and then adding in averages for travel and lodging costs to and from our area of Toronto.

https://pbs.twimg.com/media/CSgK2lAW4AUm1Pa.png:large

As a conference organizer I know that talks take time to prepare but I also know that I need to have a budget and stick to it to be successful, and that means finding a way to equally pay speakers.

I am very open to everyone input on how to analyze the data and develop a standard for base pay. Pay in itself is also variable you may pay more for popularity etc, but would be nice to have a baseline.

**Comment by fullstackto** [comment]
Oh and thanks for the break down @kosamari of one of your presentation, but what do you mean by a "full day" is that around 8hours, 5 hours, 24hours? Breaks, interruptions?

**Comment by fullstackto** [comment]
Doing an exact conversion of

1 day = 8 hours
1 week = 5 days
1 month = 4 week

The average was 66 hours

So the question is do you believe that 66hours is an accurate average for the time it takes to prepare a 1 hour presentation?

If you take an example of a teacher they've been said to spend 1 hour for each 1 hour class.

**Comment by fullstackto** [comment]
Average talk length was 66minutes.

66 hours to do a 66 minute presentation, 1 hours per minute.

How much can be cut off due to it being your hobby and you would do it any ways? or how much for this being content you would re-use else where, how often do you present the same content?

A lot of questions that I would love help to come up with a more accurate baseline of how long a 60 minute presentation takes to make.

**Comment by fullstackto** [comment]
Grading it on a curve resulted in 30hrs for 60 minutes, at that many hours I wouldn't use a freelance rate anymore but a full time job substitution rate at 75k a year before taxes thats $35/hr resulting in a base rate of 1050, current base was set at 750.

**Comment by KatieK2** [comment]
@fullstackto 

I estimate that it took me 155 hours to prepare for my most recent talk. I've given it only once, and I currently don't have any events lined up re-use it. (I've only ever re-used "half" a talk - I expanded a lightning talk into a full-size one.) 

Talk preparation time varies _hugely_ - it varies not just by person but by the topic. Because:
- Some talks are more research-based.
- Some talks are more story-based.
- Some talks have demos.
- Some demos are hardware.
- Some demos have bugs.
- Some browsers have bugs.
- Some people practice a lot.
- Some people have to plan their own travel to the conference. (Try booking international flights with layovers and a hotel room under budget in less than an hour.)

You might be able to collect a more accurate data set with another survey and a sufficiently large number of respondents. But I don't think you'll be able to get an accurate hourly count out of _this_ project; it's just not set up for that.

FWIW, I don't expect to be paid to speak at a non-profit conference; I do it for the love of the community. But I do expect the organizers to respect the huge amount of time I invest into creating content for them, and to help me be the best speaker that I can be by the time I step on stage.

**Comment by fullstackto** [comment]
I agree fully @KatieK2 this data set however is the first to even remotely touch on how much people are being paid, so from it tried to develop a baseline number because we do respect the effort it takes to making a good talk, we as an organization also accept a differnent calibre of talks that showcase the WHY and HOW of an idea, have a demo, code sample, or case study attached to them, and are 45 minutes long with 15mins of QA, we call it "workshop style" not exactly a workshop but close to it.

We then bring in 45 of these for our event! 

If I go by the direct 1minute = 1 hour then I covert the idea of how to pay from a freelance gig to a full time gig and assume a base salary of $75,000/yr making a 60 minute presentation cost $2160, factoring in variability in what the data meant by days, hours, months etc, I feel safe halving that for a median number that a conference should pay at least $1000 per talk.

And then subsidize the rest of the 1160 with travel and lodging assistance, it's very easy as an organizer to hire a travel agent to handle everyones bookings if we're the ones paying for it.
